corehq/apps/userreports/transforms/custom/date.py

```diff
@@ -1,5 +1,13 @@
 import calendar
+from datetime import datetime
+from dimagi.utils.dates import force_to_datetime
 
 
 def get_month_display(month_index):
     return calendar.month_name[int(month_index)]
+
+
+def days_elapsed_from_date(date):
+    date = force_to_datetime(date)
+    now = datetime.utcnow()
+    return (now - date).days
```

Comment:
I think there are some problems with this approach. In particular, it assumes that all UCRs will be processed in ~real time, but if you are reindexing a new report then the current delta will be way off from the delta when the form was submitted.

I think what you need to do is date math - and the anchor date should be `received_on`, no? 

I don't think we can do any indicator that depends on the current time in a meaningful way.